README.md

```diff
@@ -1,24 +1,82 @@
-# README
+# Private Events
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+The private-event project allows the user to create and attend events that take place at a specific date and time.
 
-Things you may want to cover:
+## Build with
+* Ruby 3.0
+* Rais 6
+* SQL lite
 
-* Ruby version
+## Getting started
 
-* System dependencies
+To clone and run the application locally, use the following steps:
+- Clone the repository from GitHub, using the `git clone` command.
+- Create the new application and switch to its folder `$cd folder name`.
```

Comment:
The people don't need to create the new app. It's already created with the `git clone`